lib/va1010_forms/utils.rb

```diff
@@ -5,11 +5,11 @@
 
 module VA1010Forms
   module Utils
-    def es_submit(parsed_form, form_id)
-      formatted = HCA::EnrollmentSystem.veteran_to_save_submit_form(parsed_form, @user, form_id)
-      content = Gyoku.xml(formatted)
-      submission = soap.build_request(:save_submit_form, message: content)
-      response = perform(:post, '', submission.body)
+    include ActionView::Helpers::NumberHelper
+    def es_submit(parsed_form, user, form_id)
```

Comment:
I refactored this to pass in the user instead of using the @user. It will make writing some specs for the util module much easier in the future, but for sake of scope I am not adding a specific spec for the `es_submit` method there atm (it is currently tested in the two services using this). Those specs are also already testing this functionality, I think this just makes it cleaner.